src/Files.App/Converters/BoolToGlyph.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Files.App.Converters
+{
+	public class RecentFilesBoolToGlyph
+	{
+		public static object Convert(bool value)
+		{
+			return value ? "\xE81C" : "\xE7C3";
```

Comment:
I should have specified, the values for the converter should be in the xaml file as a resource. It might be easier to just add a comment to the xaml file and skip using a converter all together.